app/controllers/automobiles_controller.rb

```diff
@@ -0,0 +1,9 @@
+class AutomobilesController < ApplicationController
+  def index
+    @automobiles = Automobile.all
+  end
+
+  def show
```

Comment:
you can remove this action because it is out of scope for this feature and you dont use it